app/models/fae/static_page.rb

```diff
@@ -39,7 +39,7 @@ def self.setup_dynamic_singleton
             define_validations("#{name}_#{lang}", type, value[:validates]) if value.try(:[], :validates).present?
           end
           # Save with lookup to have default language return in front-end use (don't need to worry about validations here)
-          define_association(name, type, "#{name}_#{I18n.locale}")
+          define_association(name, type, "#{name}_#{languages.first}")
```

Comment:
Instead of `languages.first` we should use the Rails default locale setting, which is more intuitive: `Rails.application.config.i18n.default_locale`